README.md

```diff
@@ -0,0 +1,4 @@
+Berlin, November 2024 
+This repository is a part of the instrument automation sofwtare tomato. 
+It is dedicated to the alicat devices. 
+The codes are written by Dr.Peter Kraus and Gbocho Alexandre. 
```

Comment:
Please follow the example of `tomato-bronkhorst`:

https://github.com/dgbowl/tomato-bronkhorst/blob/main/README.md